modules/otel-config-receive-metrics-monitoring-stack.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * otel/deploying-otel.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="configuration-for-receiving-metrics-from-the-monitoring-stack_{context}"]
+= Configuration for receiving metrics from the monitoring stack
+
+The OpenTelemetry Collector custom resource (CR) can configure Prometheus receiver to scrape metrics from the in-cluster monitoring stack.
+
+.Example of the OpenTelemetry Collector custom resource to scrape metrics from in-cluster monitoring stack.
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-monitoring-view # <1>
+subjects:
+  - kind: ServiceAccount
+    name: otel-collector
+    namespace: observability
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cabundle
+  namespce: observability
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true" # <2>
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: observability
+spec:
+  volumeMounts:
+    - name: cabundle-volume
+      mountPath: /etc/pki/ca-trust/source/service-ca
+      readOnly: true
+  volumes:
+    - name: cabundle-volume
+      configMap:
+        name: cabundle
+  mode: deployment
+  config: |
+    receivers:
+      prometheus: # <3>
+        config:
+          scrape_configs:
+            - job_name: 'federate'
+              scrape_interval: 15s
+              scheme: https
+              tls_config:
+                ca_file: /etc/pki/ca-trust/source/service-ca/service-ca.crt
+              bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+              honor_labels: true
+              params:
+                'match[]':
+                  - '{__name__="<metric_name>"}' # <4>
+              metrics_path: '/federate'
+              static_configs:
+                - targets:
+                  - "prometheus-k8s.openshift-monitoring.svc.cluster.local:9091"
+
+    exporters:
+      debug: # <5>
+        verbosity: detailed
+
+    service:
+      pipelines:
+        metrics:
+          receivers: [prometheus]
+          processors: []
+          exporters: [debug]
+----
+<1> Give OpenTelemetry collector service account `cluster-monitoring-view` cluster role to access metric data.
+<2> Inject OpenShift service CA for configuring TLS in the Prometheus receiver.
+<3> Configure Prometheus receiver to scrape the federate endpoint from in-cluster monitoring stack.
+<4> Use the Prometheus query language to select metrics which should be scraped. Refer to the in-cluster monitoring documentation for more details and limitations of the federate endpoint.
```

Comment:
@pavolloffay, would you be able to provide just a URL in reply in the comment here? I'll take care of the rest.
I tried searching the OpenShift docs and was not confident picking a link to add here.

Reply:
I put a link to the monitoring docs in the `Additional resources` section 

```
* xref:../monitoring/accessing-third-party-monitoring-apis.adoc#monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_accessing-third-party-monitoring-apis[Querying metrics by using the federation endpoint for Prometheus]
```